src/model/game/StorePlatform.ts

```diff
@@ -4,5 +4,6 @@ export enum StorePlatform {
     EPIC_GAMES_STORE = "Epic Games Store",
     OCULUS_STORE = "Oculus Store",
     ORIGIN = "Origin / EA Desktop",
+    PC_GAME_PASS = "PC Game Pass",
```

Comment:
PC_GAME_PASS should be XBOX_GAME_PASS

Reply:
Although Microsoft renamed "Xbox Game Pass for PC" to "PC Game Pass", I guess Xbox Game Pass is still a more recognizable name for players.